src/html_files/utils.ts

```diff
@@ -171,3 +173,37 @@ function allRunCPUListUnchanged(cpu_list) {
     }
     return true;
 }
+
+function get_inside_value(data) {
+    if ('F64' in data) {
+        return data['F64'];
+    } else if ('UInt64' in data) {
+        return data['UInt64'];
+    } else if ('String' in data) {
+        return data['String'];
+    } else if ('Stats' in data) {
+        if (comparator == 'mean') {
+            return data['Stats']['mean'];
```

Comment:
This is going to keep growing if we add more default aggregations, can leverage this typescript syntax to return a default if not found:
`data.get('Stats').get('p99.9') ?? undefined`

Reply:
Makes sense. I'll update all the aggregations to use this.